libs/cms/src/lib/cms.contentful.service.ts

```diff
@@ -174,36 +174,26 @@ export class CmsContentfulService {
   }
 
   async getOrganizationLogos(
-    organizationTitles: string[],
-    searchByKey?: boolean,
+    organizationFieldValues: string[],
+    searchByField: keyof types.IOrganizationFields,
   ): Promise<Array<string | null>> {
-    const fieldParam = searchByKey
-      ? 'fields.referenceIdentifier[in]'
-      : 'fields.title[in]'
     const params = {
       ['content_type']: 'organization',
-      select: 'fields.logo,fields.title,fields.referenceIdentifier',
-      [fieldParam]: organizationTitles.join(','),
+      select: `fields.logo,fields.${searchByField}`,
+      [`fields.${searchByField}[in]`]: organizationFieldValues.join(','),
     }
 
     const result = await this.contentfulRepository
       .getLocalizedEntries<types.IOrganizationFields>(null, params)
       .catch(errorHandler('getOrganizationsLogo'))
 
-    return organizationTitles.map((title) => {
+    return organizationFieldValues.map((title) => {
```

Comment:
Nitpick but maybe having .map((fieldValue) => ... would make more sense than calling this title